diff --git a/node_modules/react-chessboard/dist/index.esm.js b/node_modules/react-chessboard/dist/index.esm.js
index 2208b80..de6016e 100644
--- a/node_modules/react-chessboard/dist/index.esm.js
+++ b/node_modules/react-chessboard/dist/index.esm.js
@@ -5398,7 +5398,9 @@ const Square = memo(function Square({ children, squareId, isLightSquare, isOver,
             setIsClickingOnMobile(true);
         }, onTouchEnd: (e) => {
             // Prevent default to avoid double-firing with onClick on some devices
-            e.preventDefault();
+            if (e.cancelable) {
+                e.preventDefault();
+            }
             // we only want to fire onSquareClick if the touch event ended within the same square as it started
             const within = isTouchEndWithinSquare(id, squareId, e);
             if (within && isClickingOnMobile) {
diff --git a/node_modules/react-chessboard/dist/index.js b/node_modules/react-chessboard/dist/index.js
index b8667be..b196d1b 100644
--- a/node_modules/react-chessboard/dist/index.js
+++ b/node_modules/react-chessboard/dist/index.js
@@ -5400,7 +5400,9 @@ const Square = React.memo(function Square({ children, squareId, isLightSquare, i
             setIsClickingOnMobile(true);
         }, onTouchEnd: (e) => {
             // Prevent default to avoid double-firing with onClick on some devices
-            e.preventDefault();
+            if (e.cancelable) {
+                e.preventDefault();
+            }
             // we only want to fire onSquareClick if the touch event ended within the same square as it started
             const within = isTouchEndWithinSquare(id, squareId, e);
             if (within && isClickingOnMobile) {
